### PR TITLE
Policy typo

### PR DESCRIPTION
**Pull request by codytipton**
nodes_critical policy should trigger on nodes_critical >= 1

### CODE REVIEW

**Comment by Grumblesmith** [comment]
@codytipton I think the condition might be more accurate as >0. I think it might also be good to have the metric use the max statistic so we're alerting on whole numbers of critical nodes for the period.

**Review by rclagett — CHANGES_REQUESTED** [review]
Can you add an entry to README.md?

**Comment by codytipton** [comment]
> @codytipton I think the condition might be more accurate as >0. I think it might also be good to have the metric use the max statistic so we're alerting on whole numbers of critical nodes for the period.

@Grumblesmith I've applied your condition recommendation.  Please let me know if I properly implemented the max statistic.

**Review by yadima — COMMENTED** [review]
"metric['consul.catalog.nodes_critical'].max" ?

**Comment by Grumblesmith** [comment]
@codytipton is a separate metric needed? I think we could just set the statistic to max like we've done for [some of the others](https://github.com/netuitive-community-packages/netuitive-packages-linux-consul/blob/master/analyticConfigurations/metric_meta-consul.json#L15). Is there a scenario where you would alert on the average number of critical nodes, rather than the max?

File: analyticConfigurations/metric_meta-consul.json [hunk 1]
```diff
@@ -4,9 +4,8 @@
       {
         "match": "consul.catalog.(total|nodes|services)_.*",
         "properties": {
-          "tags": {
-            "unit": "count"
-          }
+          "statistic": "MAX",
+          "type": "COUNTER"
```
Comment (Grumblesmith) [inline analyticConfigurations/metric_meta-consul.json]: This should only be set if the sample values are ever increasing. An example of that would be a count of all transactions ever processed by a node.

File: policies/linux.consul.-.catalog.nodes_warning.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 0,
+        "metric": "consul.catalog.nodes_warning",
+        "operator": ">"
+      }
+    ],
+    "deleted": false,
+    "description": "consul.catalog.nodes_warning is greater than 0",
```
Comment (Grumblesmith) [inline policies/linux.consul.-.catalog.nodes_warning.json]: Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken?

File: policies/linux.consul.-.catalog.services_critical.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    "conditions": [
+      {
+        "analytic": "actual",
+        "level": 0,
+        "metric": "consul.catalog.services_critical",
+        "operator": ">"
+      }
+    ],
+    "deleted": false,
+    "description": "consul.catalog.services_critical is greater than 0",
```
Comment (Grumblesmith) [inline policies/linux.consul.-.catalog.services_critical.json]: Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken?

File: policies/linux.consul.-.catalog.nodes_up.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "consul.catalog.nodes_up",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a WARNING event when the number of active nodes drops.",
```
Comment (Grumblesmith) [inline policies/linux.consul.-.catalog.nodes_up.json]: Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken? Given the title of the policy, as a user I'd expect some mention of possible node loss in the description.

File: README.md [hunk 5]
```diff
@@ -6,5 +6,9 @@ A set of Netuitive analytics configurations, polices, dashboards, and reports th
 
 ### Version next
 
+- Fix operation error in nodes_critical policy, add max statistic for nodes_critical computed metric
+
```
Comment (Grumblesmith) [inline README.md:12]: Please add a readme entry for the addition of the new policies.

File: policies/linux.consul.-.catalog.total_nodes.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    ],
+    "conditions": [
+      {
+        "analytic": "baselineDeviation",
+        "metric": "consul.catalog.total_nodes",
+        "operator": "<"
+      }
+    ],
+    "deleted": false,
+    "description": "This policy will generate a WARNING event when the number of total nodes drops.",
```
Comment (Grumblesmith) [inline policies/linux.consul.-.catalog.total_nodes.json]: Could you provide a more user friendly description of what this policy is indicating to a user receiving the alert? Any action which should be taken?

File: policies/linux.consul.-.catalog.total_nodes.json [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    "duration": 900,
+    "enabled": false,
+    "name": "Consul - Total Nodes Deviation",
```
Comment (Grumblesmith) [inline policies/linux.consul.-.catalog.total_nodes.json]: This title should indicate the direction of the deviation, if there is one. Even better might be something like "Consul - Unusually Low Number of Nodes".

**Review by Grumblesmith — CHANGES_REQUESTED** [review]

File: policies/linux.consul.-.catalog.total_nodes.json (hunk 7 above)
Comment (yadima) [reply policies/linux.consul.-.catalog.total_nodes.json]: @Grumblesmith I've commited the requested changes to the branch

File: policies/linux.consul.-.catalog.total_nodes.json (hunk 6 above)
Comment (yadima) [reply policies/linux.consul.-.catalog.total_nodes.json]: @Grumblesmith I've commited the requested changes to the branch

File: README.md (hunk 5 above)
Comment (yadima) [reply README.md:12]: @Grumblesmith Done

File: policies/linux.consul.-.catalog.nodes_up.json (hunk 4 above)
Comment (yadima) [reply policies/linux.consul.-.catalog.nodes_up.json]: @Grumblesmith I've commited the requested changes to the branch

File: policies/linux.consul.-.catalog.services_critical.json (hunk 3 above)
Comment (yadima) [reply policies/linux.consul.-.catalog.services_critical.json]: @Grumblesmith I've commited the requested changes to the branch

File: policies/linux.consul.-.catalog.nodes_warning.json (hunk 2 above)
Comment (yadima) [reply policies/linux.consul.-.catalog.nodes_warning.json]: I've commited the requested changes to the branch

File: analyticConfigurations/metric_meta-consul.json (hunk 1 above)
Comment (codytipton) [reply analyticConfigurations/metric_meta-consul.json]: I've removed the "type" definition for this metric

File: README.md [hunk 8]
```diff
@@ -6,5 +6,12 @@ A set of Netuitive analytics configurations, polices, dashboards, and reports th
 
 ### Version next
 
+### Version 1.0.1
```
Comment (Grumblesmith) [inline README.md]:
```suggestion
```
Versions shouldn't be set in PRs, so this should still be Version next.

Comment (yadima) [reply README.md]: @Grumblesmith Done - I moved the description back to the Version next. Thanks.

**Review by Grumblesmith — APPROVED** [review]

**Review by rclagett — APPROVED** [review]

**Pull request merged**